### PR TITLE
Improve brainglobe-segmentation 1D segmentation documentation

### PR DESCRIPTION
**Pull request by jingjie-li**

### CODE REVIEW

**Review by adamltyson — CHANGES_REQUESTED** [review]
Thanks for this @jingjie-li, much appreciated! I've left a few comments, and some suggestions just to standardise formatting with the rest of the website.

File: docs/source/tutorials/silicon-probe-tracking.md [hunk 1]
```diff
@@ -88,4 +88,24 @@ directory, and follow the main brainglobe-segmentation instructions [here](./seg
...
+## Visualize the probe track with brainrender
+after the segmentation process, click `to brainrender` button in segmentation GUI. A [npy file](/documentation/brainglobe-segmentation/output-files) will be saved for each track. 
+
+with `brainrender`, you can load the `npy` file with the [scene class](/documentation/brainrender/usage/scene). 
+
+:::{note}
+You may want to use different [color](https://matplotlib.org/stable/users/explain/colors/colormaps.html) for each probe.
+:::
+
+For example: (package importing and rendering code can refer to [this example](https://github.com/brainglobe/brainrender/blob/main/examples/add_cells.py))
```
[the remark this answers is not in the document]
Comment (jingjie-li) [reply docs/source/tutorials/silicon-probe-tracking.md:102]: Yes it's better to have a full example here. but it's a bit strange to put that much code in the doc. And it need the `.npy` file to run. Can I put those as an example in `brainrender` repo and also attached the example data file and then attach that link here?

File: docs/source/documentation/brainglobe-segmentation/user-guide/output-files.md
[the remark this answers is not in the document]
Comment (jingjie-li) [reply docs/source/documentation/brainglobe-segmentation/user-guide/output-files.md:1]: yes I have moved it to tutorial.

File: docs/source/documentation/brainglobe-segmentation/user-guide/output-files.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Output files
+
+## 1D segmentation
+all files will be saved in `reg_result_path/segmentation/atlas_space/tracks` if you loaded the data from atlas space, otherwise, it will be in the folder of `sample_space`.
+
+three files will be saved for each 1D tracks:
++ `xxx.csv` a csv table that summrize the depth, brain area, brain region ID (based on allen atlas) for each point of the fitted spline. [example](https://github.com/brainglobe/brainglobe-segmentation/blob/main/tests/data/brainreg_output/segmentation/atlas_space/tracks/test_track.csv).
```
[the remark this answers is not in the document]
Comment (jingjie-li) [reply docs/source/documentation/brainglobe-segmentation/user-guide/output-files.md]: got it

File: docs/source/tutorials/silicon-probe-tracking.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (adamltyson) [reply docs/source/tutorials/silicon-probe-tracking.md:101]: I'm not sure it's so strange. I think it's something that lots of people will want to do. You could either add it to the brainrender examples along with the `.npy` file (feel free to raise a PR), or just add it here without the data making it obvious where to add the path to the file (e.g. `.path/to/brainrender_export.npy`).

[the remark this answers is not in the document]
Comment (jingjie-li) [reply docs/source/tutorials/silicon-probe-tracking.md]: I have linked this example to `brainrender` repo, here is the PR: https://github.com/brainglobe/brainrender/pull/313

**Review by adamltyson — APPROVED** [review]
Great, thanks @jingjie-li. I made some small changes but I think this is good to go :tada:

**Pull request merged**